packages/medusa-plugin-sendgrid/src/services/sendgrid.js

```diff
@@ -1155,6 +1163,24 @@ class SendGridService extends NotificationService {
     return data
   }
 
+  async orderRefundCreatedData({ refund_id }) {
+    const refund = await this.paymentProviderService.retrieveRefund(refund_id, {
+      select: [
+        'id',
+        'amount',
+        'reason',
+      ],
+      relations: [
+        'order'
+      ]
+    })
+
+    return {
+      refund,
+      email: refund.order.email
+    }
+  }
```

Comment:
suggestion: Instead of only sending the refund data + email to SendGrid, I suggest we use the `OrderService` (which is already a dependency) to find the correct refund and return them both. That would allow the consumer to add additional useful information in their email templates. Let me know what you think!

Reply:
@olivermrbl Thanks for taking a look at my PR and for the suggestion!

I like that idea because I think providing the most usable data to sendgrid that we can is a good idea. I just need a little clarification on your suggestion here though, when you say:

> I suggest we use the OrderService (which is already a dependency) to find the correct refund and return them both. 

Did you mean to find the correct order and then return the refund and the order?

Reply:
Yes exactly - sorry!

So something like:
```
   async orderRefundCreatedData({ id, refund_id }) {
    const order = await this.orderService_.retrieveWithTotals(id, {
      select: [
        ...
      ],
      relations: [
        "refunds"
      ]
    })

    const refund = order.refunds.find(refund => refund.id === refund_id)

    return {
      refund,
      order
    }
  }
```

Reply:
@olivermrbl sorry for the slight delay on this, but this looks good! I was able to make these changes so you can check them out. 

I also added a formatted `refund_amount` so that it can be used in the template since there is no good way to do that in sendgrid.